bioimageio/spec/commands.py

```diff
@@ -36,11 +36,9 @@ def validate(
 
             if is_path:
                 rdf_source = pathlib.Path(rdf_source)
-            else:
-                raise RuntimeError(f"Could not retrieve {rdf_source}")
 
         if yaml is None:
-            raise RuntimeError("Cannot validate from file without ruamel.yaml dependency!")
+            raise RuntimeError("Cannot validate from file or yaml string without ruamel.yaml dependency!")
 
         rdf_source = yaml.load(rdf_source)
```

Comment:
Does this really work? Usually the syntax would be `yaml.loads(rdf_source)` if this is a string encoded dict.
(I am not so sure about yaml though, but this is the python default way of doing it, e.g. for pickle or json)

Reply:
`yaml.load(rdf_source)` only works for ruamel.yaml's `yaml=YAML(typ="safe")` instance, but since that's what we are using it should be good to go...

Reply:
... (given that rdf_source) is a yaml encoded string

Reply:
Ok, guess ruamel choose a bit of a weird API